packages/api-server/src/api/auth.ts

```diff
@@ -30,7 +30,7 @@ router.get('/callback', async (req, res) => {
       throw new Error('NoRedirect')
     }
 
-    const subdomain = isProd ? req.cookies.subdomain : process.env.HOST
+    const path = isProd ? req.cookies.path : process.env.HOST
```

Comment:
I think you can keep this for now, but eventually this will be irrelevant. You can just redirect them back to the right path. This was made because I had have a different callback domain to auth someone, where as the path based won't have that issue since it'll be made on the same domain instead of some subdomain. Everything was done on `auth.portaler.zone` but then you'd be redirected with this cookie to `yourguild.portaler.zone`

This is one of the reasons subdomains really fucked me.